src/Laravel/Eloquent/PropertyAccess/PropertyAccessor.php

```diff
@@ -22,9 +23,12 @@
  */
 final class PropertyAccessor implements PropertyAccessorInterface
 {
+    private readonly PropertyAccessorInterface $inner;
+
     public function __construct(
-        private readonly PropertyAccessorInterface $inner,
+        ?PropertyAccessorInterface $inner = null,
```

Comment:
```suggestion
        private readonly ?PropertyAccessorInterface $inner =  PropertyAccess::createPropertyAccessor(),
```

Reply:
This syntax isn't supported (`expression is not allowed as parameter default value`) and readonly classes aren't available in 8.1.